docs-chef-io/content/inspec/resources/k8s_namespace.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "k8s_namespace resource"
+draft = false
+gh_repo = "inspec"
+platform = "k8s"
+
+[menu]
+[menu.inspec]
+title = "k8s_namespace"
+identifier = "inspec/resources/k8s/K8s Namespace"
+parent = "inspec/resources/k8s"
++++
+
+
+Use the `k8s_namespace` Chef InSpec audit resource to test the configuration of a specific namespace.
+
+## Installation
+
+## Syntax
+
+```ruby
+describe k8s_namespace(name: 'default') do
+  it { should exist }
+end
+```
+
+## Parameter
+
+`name`
+: Name of the Namespace.
+
+## Properties
+
+`uid`
+: UID of the Namespace.
+
+`name`
+: Name of the Namespace.
+
+`resource_version`
+: Resource version of the Namespace. This is an alias of `resourceVersion`.
+
+`labels`
+: Labels associated with the Namespace.
+
+`kind`
+: Resource type of the Namespace.
+
+`creation_timestamp`
+: Creation time of the Namespace. This is an alias of `creationTimestamp`.
+
+`metadata`
+: Metadata for the Namespace.
+
+## Examples
+
+### Given namespace must exist and test its properties
+
+```ruby
+describe k8s_namespace(name: 'kube-node-lease') do
+  it { should exist }
+  its('uid') { should eq '5ed76d62-838b-45cb-b41f-789b567a2fa2' }
+  its('name') { should eq 'kube-node-lease' }
+  its('kind') { should eq 'Namespace' }
+  its('resource_version') { should eq '6' }
+  its('creationTimestamp') { should eq '2022-07-21T10:47:49Z' }
```

Comment:
In the deployment resource you aliased the creation_timestamp do we really need to document the `creationTimestamp` https://github.com/inspec/inspec-k8s/pull/15/files#diff-a68e95629f978fb5d2fe91ef51e1b2a14fdc8fabbf4b906cac4f42edf07209b3R51

Reply:
Imo the user should be able to use either `creation_timestamp` or `creationTimestamp`